doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
 
-We will use the [Azure SDK Generated Client template](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg) project as a reference. We have a template [swagger file](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/template-dpg/Azure.Template.Generated/src/swagger) containing a REST API definition based on [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). We will use the swagger to generate SDK using [autorest.csharp](https://github.com/Azure/autorest.csharp).
+We will use the swagger file which contains a REST API definition based on [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). We will use the swagger to generate SDK using [autorest.csharp](https://github.com/Azure/autorest.csharp).
 
 **Learn more**: to understand more about Azure SDK Code Geneneration, see the [Azure SDK Code Geneneration docs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md).
 
 This tutorial has following sections:
 
-- [Prerequisites](#prerequisites)
-- [Setup your repo](#setup-your-repo)
-- [Getting started](#getting-started)
-  - [Create the project](#create-the-project)
-  - [Create the generated layer](#create-the-generated-layer)
-- [Tests](#tests)
-- [Samples](#samples)
-- [Snippets](#snippets)
-- [Export Public API](#export-public-api)
-- [README](#readme)
-- [Changelog](#changelog)
+- [Azure SDK Code Geneneration Quickstart Tutorial (Data Plane)](#azure-sdk-code-geneneration-quickstart-tutorial-data-plane)
+  - [Prerequisites](#prerequisites)
+  - [Setup your repo](#setup-your-repo)
+  - [Getting started - shippable package](#getting-started---shippable-package)
+    - [Create the project folder](#create-the-project-folder)
+      - [1. install dotnet tempate](#1-install-dotnet-tempate)
+      - [2. dotnet new project](#2-dotnet-new-project)
+      - [3. update the solution file if needed](#3-update-the-solution-file-if-needed)
+    - [Generate client library](#generate-client-library)
+    - [Export Public API](#export-public-api)
+  - [Improve the SDK library package](#improve-the-sdk-library-package)
+    - [Tests](#tests)
+    - [Samples](#samples)
+    - [Snippets](#snippets)
+    - [README.md](#readmemd)
+    - [Changelog](#changelog)
 
 <!-- /TOC -->
 
 ## Prerequisites
 
-- Install VS 2019 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
+- Install VS 2020 (Community or higher) and make sure you have the [latest updates](https://www.visualstudio.com/).
```

Comment:
From https://dotnet.microsoft.com/en-us/download/dotnet-framework,  .NET framework 4.8 is the recommended version. so do we support that version or do we only support 4.6 & 4.7

Reply:
yes. we will support .Net framework 4.6 and 4.7